packages/widget/src/controllers/transfers/evm/execute.ts

```diff
@@ -28,14 +32,31 @@ export async function executeNextEvmTransaction(
       this.host.requestUpdate();
       await tx.wait();
       this.pendingEvmApprovalTransactions.shift();
+
+      transactions.push(
+        ...(this.pendingEvmApprovalTransactions as PopulatedTransaction[]),
+        this.pendingTransferTransaction as PopulatedTransaction
```

Comment:
i see my comment about updating type assertion lead to more type assertions as `ethers` types aren't mutually completely overlapping 😅

Reply:
🤷